transforms/universal/profiler/kfp_ray/Makefile

```diff
@@ -29,8 +29,6 @@ image::
 
 load-image::
 
-set-versions: workflow-reconcile-requirements
```

Comment:
Please rebase, this should be fixed now in dev branch. thanks

Reply:
done